app/src/main/java/com/it/hungvt/movieapp/screen/home/HomePresenter.java

```diff
@@ -1,16 +1,20 @@
 package com.it.hungvt.movieapp.screen.home;
 
+import com.it.hungvt.movieapp.data.source.MovieRepository;
+
 /**
  * Listens to user actions from the UI ({@link HomeActivity}), retrieves the data and updates
  * the UI as required.
  */
 public class HomePresenter implements HomeContract.Presenter {
     private static final String TAG = HomePresenter.class.getName();
 
-    private final HomeContract.ViewModel mViewModel;
+    private HomeContract.ViewModel mViewModel;
+    private MovieRepository mMovieRepository;
 
-    public HomePresenter(HomeContract.ViewModel viewModel) {
+    public HomePresenter(HomeContract.ViewModel viewModel, MovieRepository movieRepository) {
         mViewModel = viewModel;
+        movieRepository = movieRepository;
```

Comment:
ERROR: [STYLE][SA_LOCAL_SELF_ASSIGNMENT] Self assignment of movieRepository in new com.it.hungvt.movieapp.screen.home.HomePresenter(HomeContract$ViewModel, MovieRepository)
http://findbugs.sourceforge.net/bugDescriptions.html#SA_LOCAL_SELF_ASSIGNMENT